server/shared/database/migrations/20210125094819-add-resolved-column-to-comment.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+const TABLE_NAME = 'comment';
+const COLUMN_NAME = 'resolved';
+
+exports.up = (qi, { BOOLEAN }) => qi.addColumn(TABLE_NAME, COLUMN_NAME, {
+  type: BOOLEAN,
```

Comment:
What do you think about making this `resolvedAt` timestamp instead of Boolean. Same reasoning as for `editedAt`.